src/commands/demo/preorderForms.ts

```diff
@@ -0,0 +1,66 @@
+import PartnerApi from '~/partnerApi'
+import loadConfig from '~/config'
+
+const FORM_ID = 'registration_form_1'
+const REGISTRATION_FORM = {
+  title: 'Registration form',
+  type: 'object',
+  required: ['firstName'],
+  properties: {
+    lastName: {
+      type: 'string',
+      title: 'Last name'
+    },
+    firstName: {
+      type: 'string',
+      title: 'First name'
+    },
+    age: {
+      type: 'integer',
+      title: 'Age'
+    }
+  }
+}
+const ADD_FORM_BODY = {
+  formSchema: REGISTRATION_FORM,
+  uiSchema: {}
+}
+
+function demo(api: PartnerApi) {
+  return api
+    .putPreorderForm(FORM_ID, ADD_FORM_BODY)
+    .then(() => {
+      return api.getPreorderForm(FORM_ID)
+    })
+    .then(form => {
```

Comment:
Maybe use fat arrow by default consistently while at it (as opposed function() in other places in promise chains).

Reply:
Unfortunately, this file was not recorded as a rename. I barely touched it. Agree that we should improve our sample code!